tests/integration/targets/homebrew_services/tasks/main.yml

```diff
@@ -0,0 +1,94 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- when: ansible_facts.distribution == 'MacOSX'
+  block:
+    - name: MACOS | Find brew binary
+      command: which brew
+      register: brew_which
+
+    - name: MACOS | Get owner of brew binary
+      stat:
+        path: "{{ brew_which.stdout }}"
+      register: brew_stat
+
+    - name: Homebrew Services test block
+      become: true
+      become_user: "{{ brew_stat.stat.pw_name }}"
+      block:
+      - name: MACOS | Install black
+        community.general.homebrew:
+          name: black
+          state: present
+        become: true
+        become_user: "{{ brew_stat.stat.pw_name }}"
```

Comment:
Since you put those in the block, these lines are redundant here and in the tasks below (within the block, of course).

Reply:
*facepalm*